content/Solutions/Optimizations/Security.md

```diff
@@ -1,20 +1,36 @@
 ---
 title: "Security Recommendations"
-description: "Best practices when tuning TrueNAS to minimize security vulnerabilities."
+description: "Best practices to administrate TrueNAS securely."
 weight: 30
 aliases:
   - /core/solutions/optimizations/security/
 tags:
  - coressh
  - scalessh
+keywords:
+ - TrueNAS Security
 ---
 
 {{< toc >}}
 
-When using services on TrueNAS, especially services that allow outside connections, there are some best practices to follow to ensure your system is safe and secure.
-Several different system services are disscused in this article.
+Follow these best practices to administrate TrueNAS securely.
 
-## iSCSI
+## General Recommendations
+
```

Comment:
Is this a step procedure or a list of recommendations? Parallel construction is off on a few points.